theme/less/_body-wrapper.less

```diff
@@ -12,6 +12,30 @@ body > div
   padding: 20px 30px;
 }
 
+@font-face 
+{
+  font-family: "Source Sans Pro";
+  font-weight: 400;
+  font-style: normal;
+  src: local("Source Sans Pro Regular"), local("SourceSansPro-Regular"), url("/fonts/SourceSansPro-Regular.woff2") format("woff2");
+}
+
+@font-face 
+{
+  font-family: "Source Sans Pro";
+  font-weight: 700;
+  font-style: bold;
+  src: local("Source Sans Pro Bold"), local("SourceSansPro-Bold"), url("/fonts/SourceSansPro-Bold.woff2") format("woff2");
+}
+
+@font-face 
+{
+  font-family: "Source Sans Pro";
+  font-weight: 400;
+  font-style: italic;
```

Comment:
Wonder if nowdays browsers are smart enough that can immitate `italic` without downloading specific font for that, make sense to try.